extensions/dicom-microscopy/src/index.tsx

```diff
@@ -23,7 +23,7 @@ const MicroscopyViewport = props => {
 /**
  * You can remove any of the following modules if you don't need them.
  */
-export default {
+const extension = {
```

Comment:
Could you declare the type here please?  There is an Extension type exported that has required things like id, preRegistration etc.